README.md

```diff
@@ -35,38 +35,49 @@ go get github.com/NodeFactoryIo/vedran
 ```
 Note that if you need to do this, you probably want to add your Go bin directory to your $PATH to make things easier!
 
-## Usage
+## Starting loadbalancer
 
-Load balancer is started using command `start`.
+First download latest prebuilt binaries [from releases](https://github.com/NodeFactoryIo/vedran/releases).
 
-```
-$ vedran -h
+Load balancer is started by invoking **start** command.
 
-vedran is a command line interface for polkadot load balancer
+For example `vedran-linux-amd64 start --auth-secret=supersecret`.
```

Comment:
we should probably archive binaries, so when extracted, they should all be called **vedran**

Reply:
```suggestion
For example `./vedran-linux-amd64 start --auth-secret=supersecret`.
```

Reply:
:+1: